.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,92 @@
+name: CI/CD
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Install golangci-lint
+      uses: golangci/golangci-lint-action@master
+      with:
+        version: v1.33.2
+        args: --version
```

Comment:
Perhaps add a comment at the end of this line:

> `# Don't actually run lint. We want it installed, but run it ourselves from make`

Reply:
done.